generic/debimage-generic.yaml

```diff
@@ -0,0 +1,72 @@
+{{- $architecture := or .architecture "amd64" -}}
+{{- $image := or .image "debian-generic.img" -}}
+{{- $suite := or .suite "buster" -}}
+
+architecture: {{$architecture}}
+
+actions:
+
+  - action: debootstrap
+    suite: "{{$suite}}"
+    components:
+      - main
+    mirror: https://deb.debian.org/debian
+    variant: minbase
+
+  - action: apt
+    packages: [adduser, sudo, systemd-sysv]
+
+  - action: run
+    chroot: true
+    description: Set hostname
+    command: echo "{{$architecture}}" > /etc/hostname
+    
+  - action: run
+    description: Set up user
+    chroot: true
+    command: >
+      /sbin/adduser --gecos user --disabled-password --shell /bin/bash user
+      && /sbin/adduser user sudo
+      && sh -c "echo 'user:user' | chpasswd"
+
+  - action: apt
+    packages: [
+{{if eq $architecture "i386" "amd64"}}
+      grub-pc,
```

Comment:
Please add support for EFI machines

Reply:
maybe this one will be enough
https://github.com/go-debos/debos-recipes/pull/21#pullrequestreview-600453298